docs/docs/basil.md

```diff
@@ -21,6 +21,7 @@ First we will need to [install Node.js following the instructions from their web
 ```sh
 mkdir basil
 cd basil
+truffle init
```

Comment:
Let's avoid this line here since we are planning to start the project with zos instead